docs/resources/flow.md

```diff
@@ -96,12 +96,14 @@ resource "sym_flow" "this" {
       type     = "string"
       required = true
       label    = "Reason"
+      visible  = true
     }
 
     prompt_field {
       name           = "urgency"
       type           = "string"
       required       = true
+      visible        = true
```

Comment:
Ah this is unrelated to your change, but I noticed the `get_approvers` reducer below is incorrect. It says `get_approvers(request)` but we enforce the parameter to be `event` or `evt`. Do you mind updating it to be `get_approvers(event)` while you're here? 🙇